content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -92,6 +92,18 @@ Common forms of values include:
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### kubectl.kubernetes.io/restartedAt
+
+Example: `kubectl.kubernetes.io/restartedAt: "2023-03-31T15:30:00Z"`
+
+Used on: Pod
+
+In this example, the annotation indicates that the Pod was most recently restarted at 3:30 PM UTC on March 31st, 2023.
+
+This annotation is used by kubernetes to keep track of the timestamp of the most recent container restart for a particular pod. This information can be useful for debugging and monitoring purposes.
+
+The `"kubectl.kubernetes.io/restartedAt"` annotation is automatically added to a Pod's metadata by the kubectl when the Pod is restarted. It records the time that the most recent restart of the Pod occurred, as a Unix timestamp.
```

Comment:
As aforementioned, this can be applied to other workload objects. If the tech review confirms that this is indeed the case, we need to generalize the statement and not make it specific to a pod.

```suggestion
The `"kubectl.kubernetes.io/restartedAt"` annotation is automatically added to the spec template by kubectl when the workload is restarted. It records the time of the most recent workload restart as a Unix timestamp.
```